### ISSUE TITLE
No longer updating

### ISSUE THREAD

**Issue by banesullivan** [conda-forge/pyvista-feedstock#62]
@conda-forge/help-python, this feedstock hasn't been auto-updating for the last few releases.

0.34.0 and 0.34.1 were both manually created in #58 and #59 respectively. [0.34.2 is on PyPI](https://pypi.org/project/pyvista/0.34.2/), yet this feedstock is not getting updated.

Any help would be greatly appreciated!!

**Comment by BastianZim** [comment]
Have you checked for it on https://conda-forge.org/status/

**Comment by banesullivan** [comment]
Thanks for the link - didn't know about that.

I'm still confused though. I see that [autotick bot](https://github.com/regro/autotick-bot) is failing at the moment, but this repo has been failing to get updated since April while other feedstocks I manage have had no issues in that time frame.

**Comment by BastianZim** [comment]
You can get errors for your feedstock under Version Updates. Your feedstock should be listed under “errored” there.

**Comment by banesullivan** [comment]
Thanks. Looks like this is the error:

```
The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!

Please check the URLs in your recipe with version '0.35.0' to make sure they exist!

We also found the following errors:

 - could not hash URL template 'https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz'
```

**Comment by banesullivan** [comment]
> Please check the URLs in your recipe with version '0.35.0' to make sure they exist!

0.35.0 doesn't exist on PyPI: https://pypi.org/project/pyvista/#history

> could not hash URL template 'https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz'

This URL definitely works for me. For example https://files.pythonhosted.org/packages/source/p/pyvista/pyvista-0.34.1.tar.gz

cc @akaszynski, did we accidentally release and yank version 0.35.0 and perhaps the autotick bot has been erroring since?

**Comment by akaszynski** [comment]
Yes, my bad. Back when I made the v0.34.0 release I mistagged as v0.35.0

**Comment by banesullivan** [comment]
@conda-forge/help-python, is there a way you all can help us reset the autotick bot as it seems it's hung on a failed release from back in April. Will we need to skip 0.35.0 and jump to 0.36 due to this error?

**Comment by BastianZim** [comment]
Easiest way is to manually update it and then the bot will do the next release again automatically.

**Comment by banesullivan** [comment]
@BastianZim, we've done that twice now. ^^^

https://github.com/conda-forge/pyvista-feedstock/issues/62#issue-1292160929:
> 0.34.0 and 0.34.1 were both manually created in #58 and #59 respectively. [0.34.2 is on PyPI](https://pypi.org/project/pyvista/0.34.2/), yet this feedstock is not getting updated.

**Comment by BastianZim** [comment]
Hmm strange. I would recommend using Grayskull for the meta.yaml (and adding the missing parts, Grayskull is not perfect, manually) then you're definitely sure that the meta.yaml is correct. 
If that still doesn't work it's probably best to open an issue in the bot repo.

**Comment by banesullivan** [comment]
Thanks for the tips! I'll recreate the recipe with grayskull and go from there

**Comment by dopplershift** [comment]
I'm wondering if 0.35 is still hindering it finding 0.34.1, since it's looking for things past 0.35. I'd just do 0.34.2 manually and revisit once >0.35 is released.

**Comment by banesullivan** [comment]
That's what I've been wondering. 0.35 is coming soon so hopefully after that it will resolve itself. 

-----

Also... in PyVista, we keep release branches in order to back port patches. 

E.g, we may release 0.34.99 on PyPI with a patch after 0.35.1 is released

Are these back-patches not supported on conda-forge?

**Comment by BastianZim** [comment]
Backpatches are supported, the bot just won't pick them up so you have to do them manually. 

Some more info: https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-several-versions

**Comment by banesullivan** [comment]
Looks like this has resolved now that 0.35.1 is out! Thanks for the help, all!

Ref #64
